src/applications/mhv-secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -448,9 +461,13 @@ const ComposeForm = props => {
       };
       if (
         (messageValid && !isSignatureRequired) ||
-        (isSignatureRequired && messageValid && saveError !== null)
+        (isSignatureRequired &&
+          messageValid &&
+          saveError !== null &&
+          savedDraft)
```

Comment:
Vic, would you please help me understand what adding `savedDraft` to this validation does? Does it mean if the draft was not previously saved it will not trigger save call?

Reply:
@oleksii-morgun  -- Adding `savedDraft` here checks if `Save draft` btn was clicked and to prevent the `manual saveDraftHandler` from saving in the background while the modal is visible. Once the modal is closed `setSetErrorModal(false)`, `savedDraft` is set to `false` and the draft can continue saving depending on the selection.